docs/source/interactive/python-ipython-diff.rst

```diff
@@ -89,8 +89,8 @@ shortcut to get help. A question mark alone will bring up the IPython help:
     -------------
     ...
 
-A single question mark before, or after an object available in current
-namespace will show help relative to this object:
+A single question mark before or after an object available in the current
+name-space will show help relative to this object:
```

Comment:
`namespace` is a perfectly valid word in the context of computing, there's no need to change it.

https://en.wikipedia.org/wiki/Namespace#:~:text=In%20computing%2C%20a%20namespace%20is,they%20can%20be%20easily%20identified.